src/components/LayoutTree.tsx

```diff
@@ -881,6 +881,10 @@ const LayoutTree = () => {
   return (
     <div
       className={css({
+        animation:
+          indentDepth % 2 === 0
+            ? 'hideCaretEven {durations.layoutSlowShiftDuration} linear'
+            : 'hideCaretOdd {durations.layoutSlowShiftDuration} linear',
```

Comment:
This is a gotcha with PandaCSS that unfortunately is very easy to make. (It took me a while to even figure out why the styles were not being applied on my end!)

## ✗ 

String template can only be determined at runtime, so PandaCSS is unable to statically analyze it:

```ts
animation: `${indentDepth % 2 === 0 ? 'hideCaretEven' : 'hideCaretOdd'} {durations.layoutSlowShiftDuration} linear`,
```

## ✓ 

Individual branches of the ternary are statically analyzable, so this will work:

```ts
	animation: indentDepth % 2 === 0 
	  ? 'hideCaretEven {durations.layoutSlowShiftDuration} linear' 
	  : 'hideCaretOdd {durations.layoutSlowShiftDuration} linear'
```

## ✓ 

The shorthand animation can also be broken out into each component. Similar to above, `animationName` works because individual branches of the ternary are statically analyzable:


```ts
	animationDuration: '{durations.layoutSlowShiftDuration}',
	animationName: indentDepth % 2 ? 'hideCaretEven' : 'hideCaretOdd',
	animationTimingFunction: 'linear',
```

Reply:
Right, thank you, I think I read that in the docs and then forgot about it.